draft-ietf-tls-esni.md

```diff
@@ -456,6 +456,14 @@ The client then constructs the ClientHelloInner message just as it does a
 standard ClientHello, with the exception of the following rules:
 
 1. It MUST NOT offer to negotiate TLS 1.2 or below.
```

Comment:
An editorial suggestion: I would have the bullet simply read "It MUST ensure that all HelloRetryRequest-sensitive parameters applicable to TLS 1.3 and higher versions in ClientHelloInner match that in ClientHelloOuter. See {{hrr}} for the list of these parameters." and I would enumerate the HRR-sensitive parameters in {{hrr}}. Moreover, this change likely impacts the text in {{hrr}}.

Reply:
Should be done!